postprocessing/science/compute_diff_seissol_data.py

```diff
@@ -167,7 +167,19 @@ def same_geometry(sx1, sx2):
     args.idt = list(range(0, sx1.ndt))
 
 aData = []
-for dataname in args.Data:
+if args.Data == ["all"]:
+    variable_names = set()
+    for elem in sx1.tree.iter():
+        if elem.tag == "Attribute":
+            variable_names.add(elem.get("Name"))
+    for elem in sx2.tree.iter():
+        if elem.tag == "Attribute":
+            variable_names.add(elem.get("Name"))
+    variable_names.remove("partition")
```

Comment:
also locationFlag

Reply:
I'll just take the improved version from the master branch ;-)